app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,10 @@
+class CommentsController < ProjectScopedController
```

Comment:
Let's not keep digging this hole.

include a new `ProjectScoped` controller concern (which would be empty at this stage) instead.